CRISPResso2/CRISPRessoPooledCORE.py

```diff
@@ -1554,8 +1549,8 @@ def rreplace(s, old, new):
 
         #if many reads weren't aligned, print those out for the user
         if RUNNING_MODE != 'ONLY_GENOME':
-            #N_READS_INPUT=get_n_reads_fastq(args.fastq_r1)
```

Comment:
Were these lines changed using a find and replace? Does anyone know why they are commented out and if they should just be removed?

Reply:
@trevormartinj7 @Colelyman

Reply:
These commented out lines are now deleted!